src/test/javascript/spec/component/exam/manage/exams/exam-checklist.component.spec.ts

```diff
@@ -0,0 +1,215 @@
+    const responseExerciseGroup = { body: exerciseGroupsExercisePointsEqual } as HttpResponse<ExerciseGroup[]>;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [
+                RouterTestingModule.withRoutes([
+                    { path: 'course-management/:courseId/exams/:examId/edit', component: DummyComponent },
+                    { path: 'course-management/:courseId/exams/:examId/exercise-groups', component: DummyComponent },
+                    { path: 'course-management/:courseId/exams/:examId/tutor-exam-dashboard', component: DummyComponent },
+                    { path: 'course-management/:courseId/exams/:examId/scores', component: DummyComponent },
+                    { path: 'course-management/:courseId/exams/:examId/student-exams', component: DummyComponent },
+                    { path: 'course-management/:courseId/exams/:examId/test-runs', component: DummyComponent },
+                    { path: 'course-management/:courseId/exams/:examId/students', component: DummyComponent },
+                ]),
+                HttpClientTestingModule,
+            ],
+            declarations: [
+                ExamChecklistComponent,
+                DummyComponent,
+                MockPipe(TranslatePipe),
+                MockPipe(ArtemisDatePipe),
+                MockComponent(AlertComponent),
+                MockComponent(AlertErrorComponent),
+                MockDirective(JhiTranslateDirective),
+                MockDirective(HasAnyAuthorityDirective),
+                ExamChecklistCheckComponent,
+                ExamChecklistExerciseGroupTableComponent,
+                ProgressBarComponent,
+                MockDirective(NgbTooltip),
+                MockComponent(FaIconComponent),
+            ],
+            providers: [
+                {
+                    provide: ActivatedRoute,
+                    useValue: {
+                        data: {
+                            subscribe: (fn: (value: Data) => void) =>
+                                fn({
+                                    exam,
+                                }),
+                        },
+                        snapshot: {},
+                    },
+                },
+                MockProvider(AccountService, {
+                    isAtLeastInstructorInCourse: () => true,
+                }),
+            ],
+            schemas: [],
+        })
+            .compileComponents()
+            .then(() => {
+                examChecklistComponentFixture = TestBed.createComponent(ExamChecklistComponent);
+                examDetailComponent = examChecklistComponentFixture.componentInstance;
+                exerciseGroupService = TestBed.inject(ExerciseGroupService);
+                findAllForExamStub = stub(exerciseGroupService, 'findAllForExam');
+                findAllForExamStub.returns(of(responseExerciseGroup));
+            });
+    });
+
+    beforeEach(function () {
+        // reset exam
```

Comment:
Function with a proper name instead of a comment would be better.

Reply:
As far as a now beforeEach functions within jest can't be named so I guess its okay in that case.